detection-rules/link_discord_notification.yml

```
@@ -0,0 +1,73 @@
+name: "Brand impersonation: Discord notification"
+description: "Detects inbound messages that impersonate Discord's notification system through display name spoofing, domain lookalikes, or logo usage in attachments. The messages contain typical Discord-style notification language in the subject line while failing authentication checks."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    strings.ilike(sender.display_name, '*discord*')
+    or strings.ilevenshtein(sender.display_name, 'discord') <= 2
+    or strings.ilike(sender.email.domain.domain, '*discord*')
+    // Discord logo detection in image attachments
+    or any(attachments,
+           .file_type in $file_types_images
+           and any(ml.logo_detect(.).brands,
+                   strings.starts_with(.name, "Discord")
+           )
+    )
+    // Discord logo detection in message screenshot
+    or any(ml.logo_detect(file.message_screenshot()).brands,
+           strings.starts_with(.name, "Discord") and .confidence != "low"
+    )
+  )
+  and (
+    regex.icontains(subject.subject, 'you have received a new notification')
+    or regex.icontains(subject.subject,
+                       'new (?:message|notification|alert|activity)'
+    )
+    or regex.icontains(subject.subject,
+                       '(?:message|notification|alert|activity) (?:from|waiting|pending)'
+    )
+    or regex.icontains(subject.subject, 'unread (?:message|notification)s?')
+    or regex.icontains(subject.subject,
+                       '(?:missed|pending) (?:message|notification|call)s?'
+    )
```

we can combine all of these, I think this encompasses them all? might double check there. 
```suggestion
    or regex.icontains(subject.subject,
                       '(?:new|unread|missed|pending|discord)\s+(?:message|notification|alert|activity|call)'
                       '(?:message|notification|alert|activity|call) (?:from|waiting|pending)',
                       
    )
```